$OpenBSD: patch-bindings_perl_src_CMakeLists_txt,v 1.4 2016/01/19 16:16:38 ajacoutot Exp $

From FreeBSD:
* Explicitly against libperl.so.x.y.
  This is being discussed in https://github.com/libproxy/libproxy/pull/7.
  We need this for `make stage-qa' to pass, as the Libproxy.so binding uses
  libperl symbols and thus should link against it.

--- bindings/perl/src/CMakeLists.txt.orig	Tue Jan 12 17:41:07 2016
+++ bindings/perl/src/CMakeLists.txt	Tue Jan 19 16:45:52 2016
@@ -12,7 +12,7 @@ set(Libproxy_LIB_SRCS Libproxy.c)
 set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/perl/blib/arch/auto/Net)
 add_library(PLlibproxy SHARED ${Libproxy_LIB_SRCS})
 
-target_link_libraries(PLlibproxy libproxy pthread)
+target_link_libraries(PLlibproxy perl libproxy pthread)
 set_target_properties(PLlibproxy PROPERTIES OUTPUT_NAME "Libproxy")
 set_target_properties(PLlibproxy PROPERTIES PREFIX "")
 
